apps/web/src/components/Home/SeeThroughLens.tsx

```diff
@@ -110,7 +110,7 @@ const SeeThroughLens: FC = () => {
       >
         <Menu.Items
           static
-          className="absolute w-64 right-0 z-[5] mt-1 bg-white rounded-xl border shadow-sm dark:bg-gray-900 focus:outline-none dark:border-gray-700/80"
+          className="absolute w-64 right-0 max-[412px]:left-0 z-[5] mt-1 bg-white rounded-xl border shadow-sm dark:bg-gray-900 focus:outline-none dark:border-gray-700/80"
```

Comment:
**nitpick:** we don't recommend to use hardcoded pixels 🙈

Reply:
Can you suggest better approach? 

I will surely go with that way

Reply:
I searched for better approach
And only found this. 


![Screenshot_2022-12-06-20-24-53-276-edit_com android chrome](https://user-images.githubusercontent.com/56780589/206065430-2bc24dfe-57fa-4503-9a27-d4501284862b.jpg)